rails/db/schema.rb

```diff
@@ -33,16 +33,6 @@
     t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
   end
 
-  create_table "media", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
-    t.bigint "story_id"
-    t.string "media_type"
-    t.string "description"
-    t.string "name"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.index ["story_id"], name: "index_media_on_story_id"
-  end
-
```

Comment:
We probably need to keep this in the schema right? We want to be able to associate media like videos with stories.